undefined_syms.txt

```diff
@@ -404,6 +404,21 @@ D_060006D0 = 0x060006D0;
 D_06000000 = 0x06000000;
 D_06000350 = 0x06000350;
 
+// z_en_ru2
+D_060004CC = 0x060004CC;
+D_06000F20 = 0x06000F20;
+D_060022E0 = 0x060022E0;
+D_06002AE0 = 0x06002AE0;
+D_0600C700 = 0x0600C700;
+D_0600D3DC = 0x0600D3DC;
+D_0600DCAC = 0x0600DCAC;
+D_06000DE8 = 0x06000DE8;
+D_0600E630 = 0x0600E630;
+D_0600F03C = 0x0600F03C;
+D_0600F8B8 = 0x0600F8B8;
+D_80116280 = 0x80116280;
+D_80116290 = 0x80116290;
```

Comment:
These shouldn't be defined here. For shiftability it's important to only put segment addresses in `undefined_syms.txt`